jicofo/rootfs/etc/services.d/jicofo/run

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/with-contenv bash
 
-JAVA_SYS_PROPS="-Dnet.java.sip.communicator.SC_HOME_DIR_LOCATION=/ -Dnet.java.sip.communicator.SC_HOME_DIR_NAME=config -Djava.util.logging.config.file=/config/logging.properties"
```

Comment:
Have you verified that logging works as expected after removing SC_HOME_DIR_LOCATION and SC_HOME_DIR_NAME? I see *some* code that uses them to decide where to put the log file.